dashboard/app/controllers/project_versions_controller.rb

```diff
@@ -1,5 +1,5 @@
 class ProjectVersionsController < ApplicationController
-  before_action :authenticate_user!
+  before_action :authenticate_user!, except: [:project_commits]
```

Comment:
why don't we want to authenticate_user for this route?

Reply:
Oh I totally misunderstood what it did. I'll take out the exception